ast/src/parsed/display.rs

```diff
@@ -220,7 +233,11 @@ impl<T: Display> Display for FunctionStatement<T> {
                 write_regs.join(", "),
                 assignment_reg
                     .as_ref()
-                    .map(ToString::to_string)
+                    .map(|s| s
+                        .iter()
```

Comment:
I have seen and written this soo often, maybe we should have a util function for that.